libs/langchain/langchain/chat_models/vertexai.py

```diff
@@ -118,6 +140,7 @@ def _generate(
         messages: List[BaseMessage],
         stop: Optional[List[str]] = None,
         run_manager: Optional[CallbackManagerForLLMRun] = None,
+        stream: Optional[bool] = False,
```

Comment:
if we're able to remove 
```suggestion
```

Reply:
fwiw we do have it for other models, tho i agree i don't love it. also seeing default val here is wrong ,should be None

Reply:
what we shouldn't have is the instance one